src/components/Sidebar.vue

```diff
@@ -124,6 +125,16 @@ export default {
       return true;
     });
 
+    if (country !== 'India') {
+      this.groups.forEach((group) => {
+        if (group.title === 'Reports') {
+          group.items = group.items.filter(
+            (item) => !item.label.toLowerCase().includes('gst')
+          );
+        }
+      });
+    }
```

Comment:
Is there a cleaner way to do this @piyushsinghania ?

Reply:
@ankitsinghaniyaz done 💯 
Added ability to hide or show sidebar items,
this can be reused on adding further items.

Reply:
Done in a much simpler and cleaner way, thanks @ankitsinghaniyaz 💯